src/boot/efi/util.h

```diff
@@ -43,6 +43,18 @@ static inline void freep(void *p) {
 
 #define _cleanup_free_ _cleanup_(freep)
 
+static inline void erase_obj(void *p) {
+        size_t l;
+        if (!p)
+                return;
+        l = __builtin_object_size(p, 0);
+        if (l == (size_t) -1)
```

Comment:
`SIZE_MAX`?

Reply:
The gcc documentation and other usage explicitly uses this expression `(size_t) -1`, so I thought it was best to be consistent.